logs/client.go

```diff
@@ -0,0 +1,195 @@
+package logs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/solarwinds/swo-cli/version"
+)
+
+type Client struct {
+	opts       *Options
+	httpClient http.Client
+	output     *os.File
+}
+
+type Log struct {
+	Time     time.Time `json:"time"`
+	Message  string    `json:"message"`
+	Hostname string    `json:"hostname"`
+	Severity string    `json:"severity"`
+	Program  string    `json:"program"`
+}
+
+type PageInfo struct {
+	PrevPage string `json:"prevPage"`
+	NextPage string `json:"nextPage"`
+}
+
+type LogsData struct {
+	Logs     []Log `json:"logs"`
+	PageInfo `json:"pageInfo"`
+}
+
+func NewClient(opts *Options) (*Client, error) {
+	return &Client{
+		httpClient: *http.DefaultClient,
+		opts:       opts,
+		output:     os.Stdout,
+	}, nil
+}
+
+func (c *Client) prepareRequest(ctx context.Context, nextPage string) (*http.Request, error) {
+	var logsEndpoint string
+	var err error
+	params := url.Values{}
+	if nextPage == "" {
+		logsEndpoint, err = url.JoinPath(c.opts.ApiUrl, "v1/logs")
+		params.Add("pageSize", "1000")
```

Comment:
We'll want to bring back the `direction=forward` param, now that the API supports that.

```suggestion
		params.Add("direction", "forward")
		params.Add("pageSize", "1000")
```

Other than that, I think this all looks great! With this change, it seems to all work perfectly on dev/stg, and will work on prod after the next regularly scheduled deployment.